enhancements/node-tuning/mixed-cpu-node-plugin.md

```diff
@@ -0,0 +1,353 @@
+---
+title: mixed-cpu-node-plugin
+authors:
+  - "@Tal-or"
+reviewers: 
+  - "@MarSik"
+  - "@yanirq"
+  - "@ffromani"
+  - "@jmencak"
+  - "@mrunalp"
+  - "@dhellmann"
+approvers: 
+  - "@mrunalp"
+  - "@dhellmann"
+api-approvers:
+  - "@mrunalp"
+  - "@dhellmann"
```

Comment:
```suggestion
```

Either @deads2k or @JoelSpeed should be listed as API approver.

Reply:
Done. Thanks!